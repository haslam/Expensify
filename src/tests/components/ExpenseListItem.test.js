import React from "react";
import { shallow } from "enzyme";
import ExpenseListItem from "../../components/ExpenseListItem";
import expenses from "../fixtures/Expenses";

test('should render an ExpenseListItem', () => {
  const wrapper = shallow(<ExpenseListItem {...expenses[1]} />);
  expect(wrapper).toMatchSnapshot();
})