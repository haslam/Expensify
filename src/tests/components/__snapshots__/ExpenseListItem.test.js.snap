// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an ExpenseListItem 1`] = `
<Link
  className="list-item"
  replace={false}
  to="/edit/2"
>
  <div>
    <h3
      className="list-item__title"
    >
      Water test
    </h3>
    <span
      className="list-item__subtitle"
    >
      December 27th, 1969
       
    </span>
  </div>
  <h3
    className="list-item__amount"
  >
    $18.50
  </h3>
</Link>
`;
